utils/nerdctl/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nerdctl
+PKG_VERSION:=1.6.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_GIT_URL:=github.com/containerd/nerdctl
+PKG_GIT_REF:=v$(PKG_VERSION)
```

Comment:
Why do we need to have new variables? Please take a look how it is done for other packages.

Reply:
I started from the docker Makefile and another go package, looks like they are leftovers and I got lucky ^^;

Reply:
Yeah. The Docker Makefile is awful, though. Not very great example here.